src/util/apiUtils.js

```diff
@@ -589,7 +593,7 @@ export const injectResponseCookies = request => ([response, _, jar]) => {
 			path: cookie.path,
 			isHttpOnly: cookie.httpOnly,
 			isSameSite: false,
-			isSecure: process.env.NODE_ENV === 'production',
+			isSecure: config.isProd,
```

Comment:
The unit test mock that it was using didn't have a `server` prop so I added it with `getServer()` 👍